import.sh

```diff
@@ -4,8 +4,19 @@ set -eu
 
 rm -rf *.c internal/*
 curl -sL https://github.com/jemalloc/jemalloc/releases/download/4.0.4/jemalloc-4.0.4.tar.bz2 | tar jxf - -C internal --strip-components=1
-(cd internal && ./autogen.sh && ./configure --with-jemalloc-prefix="")
-patch -p1 < gitignore.patch
+
+# You need to manually run the following code.
+# on OSX:
+# mkdir -p darwin_includes/jemalloc/internal
+# (cd internal && ./autogen.sh && ./configure --enable-prof --with-jemalloc-prefix="")
+# cp internal/include/jemalloc/*.h darwin_includes/jemalloc
```

Comment:
Ah, `git clean -n` to the rescue. Now copying the minimum number of headers. PTAL.